packages/ethereum/Dockerfile.hardhat

```diff
@@ -1,68 +0,0 @@
-ARG HOPR_TOOLCHAIN_IMAGE=${HOPR_TOOLCHAIN_IMAGE:-hopr-toolchain:latest}
```

Comment:
Where is this image build now?

Reply:
Hmmm... For this, I can only share what I understood: as `Dockerfile.hardhat` is to put hardhat into a docker image which is used by the hopr toolchain, I guess this part should have already been replaced into anvil by you?